Automattic/wp-calypso
# Adding Jetpack Growth bundle to the pricing page

Related to https://github.com/Automattic/jetpack-roadmap/issues/1958

## Proposed Changes

* adding Jetpack Growth bundle to the pricing page
* updating card alignment to display bundle image at the top of the card
* copy updates
* removing Creator product from the purchase page

## Why are these changes being made?


* allow customers to purchase the new bundle from the Jetpack pricing page

## Testing Instructions



* open `Jetpack Cloud live` from the live branch build below
* update the URL by going to `[domain]/pricing`
* verify that the plans and their lightbox descriptions are correct

New layout:
![SCR-20241122-mpbn](https://github.com/user-attachments/assets/bb4508e0-ca97-41c2-ad3c-4733967a849a)

![SCR-20241122-ngpi](https://github.com/user-attachments/assets/bfb9e1c1-febc-43aa-bde4-526ee5aefbb1)


## Pre-merge Checklist
